src/event_pusher/mod_event_pusher_push_plugin.erl

```diff
@@ -25,15 +25,21 @@
          publish_notification/5]).
 
 
+%% callback used for filtering push notifications. A push notification is triggered for a given
```

Comment:
I've seen EDoc tags scattered throughout the MIM codebase. These three comments look edoc-ish, so if there is possible profit, perhaps a `@doc` tag would be ok here. However, I don't know what's the standard for us.